donkeycar/management/base.py

```diff
@@ -273,6 +274,59 @@ def run(self, args):
         mm.run(args, parser)
 
 
+class ShowHistogram(BaseCommand):
+
+    def parse_args(self, args):
+        parser = argparse.ArgumentParser(prog='tubhist',
+                                         usage='%(prog)s [options]')
+        parser.add_argument('--tub', nargs='+', help='paths to tubs')
+        parser.add_argument('--record', default=None,
+                            help='name of record to create histogram')
+        parser.add_argument('--out', default=None,
+                            help='path where to save histogram end with .png')
+        parsed_args = parser.parse_args(args)
+        return parsed_args
+
+    def show_histogram(self, tub_paths, record_name, out):
+        '''
+        Produce a histogram of record type frequency in the given tub
+        '''
+        from matplotlib import pyplot as plt
+        from donkeycar.parts.tub_v2 import Tub
+
+        output = out or os.path.basename(tub_paths)
+        path_list = tub_paths.split(",")
+        records = [record for path in path_list for record
+                   in Tub(path, read_only=True)]
+        df = pd.DataFrame(records)
```

Comment:
Pandas is a pretty heavy package to install on a SBC. If you can avoid it, that would be a lot nicer. We have been trying to avoid dependencies on it in the past.

Reply:
Good point, at least we can move the import into the function, one wouldn't run it on the RPi anyway as it directly prints into the screen.